charts/ai-software-templates/pipeline-install/Chart.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v2
+kubeVersion: ">= 1.27.0-0"
+annotations:
+  charts.openshift.io/name: OpenShift Pipelines Install for AI sample applications
```

We might want to use the directory name we have created for all of our helm charts in openshift charts repo.
```suggestion
  charts.openshift.io/name: OpenShift Pipelines Install for ai-lab helm charts
```